docs/streams/developer-guide/processor-api.html

```diff
@@ -75,7 +75,7 @@ <h2><a class="toc-backref" href="#id1">Overview</a><a class="headerlink" href="#
                 phase. Processor instances should perform any required initialization in this method. The <code class="docutils literal"><span class="pre">init()</span></code> method passes in a <code class="docutils literal"><span class="pre">ProcessorContext</span></code>
                 instance, which provides access to the metadata of the currently processed record, including its source Kafka topic and partition,
                 its corresponding message offset, and further such information. You can also use this context instance to schedule a punctuation
-                function (via <code class="docutils literal"><span class="pre">ProcessorContext#schedule()</span></code>), to forward a new record as a key-value pair to the downstream processors (via <code class="docutils literal"><span class="pre">ProcessorContext#forward()</span></code>),
+                function (via <code class="docutils literal"><span class="pre">ProcessorContext#schedule()</span></code>), to forward a new record as a key-value pair with an arbitrary timestamp to the downstream processors (via <code class="docutils literal"><span class="pre">ProcessorContext#forward()</span></code>),
```

Comment:
This is a bit misleading: maybe better elaborate a bit more that using `foward(k, v)` then the punctuated timestamp will be used; using `forward(k, v, to)` then use whatever specified.